Kernel/Theory.cpp

```diff
@@ -1608,19 +1600,19 @@ bool Theory::tryInterpretConstant(unsigned func, RealConstantType& res)
 
 Term* Theory::representConstant(const IntegerConstantType& num)
 {
-  unsigned func = env.signature->addIntegerConstant(num);
+  unsigned func = env.signature->addNumeralConstant(num);
```

Comment:
Would it make sense to have the below three functions become one templated function?